tweakwcs/imalign.py

```diff
@@ -282,7 +282,7 @@ def tweak_image_wcs(images, refcat=None, enforce_user_order=True,
 
             rcat = refcat.meta['catalog'].copy()
 
-            if 'RA' not in catalog.colnames or 'DEC' not in catalog.colnames:
+            if 'RA' not in rcat.colnames or 'DEC' not in rcat.colnames:
```

Comment:
@stsci-hack This is the line that is for `NDData` objects. `rcat` here has `colnames` because it is an `atropy.table.Table` object.